internal/controller/atlasschema_controller.go

```diff
@@ -132,6 +136,10 @@ func (r *AtlasSchemaReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		r.recordErrEvent(res, err)
 		return result(err)
 	}
+	if data.hasTargets() {
+		res.SetNotReady("MultipleTargets", "Multiple targets are not supported")
```

Comment:
Don't add new reason, because we need document it also keep it alive after added.
```suggestion
		res.SetNotReady("ReadSchema", "Multiple targets are not supported")
```